### DISCUSSION TITLE
Wie starte ich die Reinigung für ein Programm oder einen einzelnen Raum

### DISCUSSION THREAD

**Discussion by bruesche** [copystring/ioBroker.roborock#350] [Q&A] [answered]
Hallo, 

der Adapter funktioniert sehr gut. Datenpunkte werden alle gefüllt. Ich habe aber nur die Möglichkeit Raumreinigung zu starten, aber dann reinigt er die ganze Map ohne Auswahl. Wie kann ich einen einzelnen Raum reinigen lassen oder wie kann ich die angelegte Favoriten (in der App heissen die Programme) direkt Starten? 
Danke für Hilfe.

**Comment by Bloe0815** [comment, marked as answer]
Du musst die einzelnen Räume auf true bzw. false stellen und dann den Befehl "Section cleaning" (oder so ähnlich, bin gerade unterwegs) starten.
Das steht auch irgendwo im [WIKI](https://github.com/copystring/ioBroker.roborock/wiki)

**Reply by bruesche** [reply]
Funktioniert tadellos.

**Reply by trollvottel** [reply]
Und wie kann man raumspezifisch die Reinigungsoptionen setzen? Ich will nicht, dass das Ding überall moppt (Parkett). In der App geht das.

**Reply by copystring** [reply]
Aktuell gar nicht. Wenn du ein Issue dafür eröffnest, gucke ich mir das bei nächster Gelegenheit an. Dafür muss der Datenverkehr der App mitgeschnitten werden. Siehe hier: https://github.com/copystring/ioBroker.roborock/wiki/Sniffing-Roborock-MQTT-traffic wobei hinzugeführt werden muss, dass die aktuelle roborock app certificate pinning verwendet. Das bedeutet, dass ohne gerootet Handy oder ohne alte App kein sniffing mehr möglich ist.

**Reply by copystring** [reply]
Roborock scheint den Schabernack hier mitzubekommen und sperrt mich immer weiter aus. Früher konnte man einfach über MQTT den Datenverkehr mitschneiden. Mittlerweile geht das nur noch MITM.

**Reply by trollvottel** [reply]
Gut, dass so Smarthome Zeugs verschlüsselt wird, macht ja durchaus Sinn. Aber mein iPhone roote ich dafür nicht :)

**Reply by trollvottel** [reply]
Aber ich könnte Dir ja vorübergehend Zugriff / Credentials geben?

**Reply by copystring** [reply]
Ja, vielleicht das Wochenende nächste Woche. Müssten uns dann irgendwie verabreden.

**Reply by trollvottel** [reply]
Am besten kurzfristig vor dem WE dann. Ich krieg ja ne mail wenn Du mich erwähnst.

**Reply by copystring** [reply]
OK. Ich melde mich dann nächste Woche am WE

**Reply by tropisch** [reply]
Wie kann ich bei der Zimmerreinigung zB nur Saugen lassen?

**Reply by canim60** [reply]
Nabend, der Wiki ist leer :/, würde mich gerne reinlesen. Würde gerne wissen wo man nur Saugen bzw Wischen und Saugen einstellen kann. Wie Sieht es aus mit Q Revo Master? Kannst den auch einbinden? Kann meine Daten zu Verfügung stellen. MfG





`<html><body>

  | Failed  to execute undefined on robot undefined (unknown model): This robot  roborock.vacuum.a117 is not fully supported just yet. Contact the dev to  get this robot fully supported!
-- | --



</body>
</html>`

**Comment by sunpower10** [comment]
Hallo,
ich habe das gleiche Problem, möchte die Raumreinigung starten, kann aber die Einträge der Räume einfach nicht finden (sollte laut WIKI unter floors* sein). 

![image](https://github.com/user-attachments/assets/dfc680c3-698b-4573-b9b7-6a8a705633c1)

**Reply by sunpower10** [reply]
Habe die Antwort gerade selber gefunden, hier für alle die ein ähnliches Problem haben:
![image](https://github.com/user-attachments/assets/3d480905-dad7-4a63-ae43-5f616726c6f9)
